README.md

```diff
@@ -22,6 +23,20 @@ Now that Minerva is running, load the page.  At this point you can register an a
 software that powers Minerva.  Now that you are logged in with the admin user, go to the Admin console,
 then the Plugins page, enable the Minerva plugin and restart the server.  Once the server is restarted, return to the root page in the browser address bar and refresh the page.  You should see the Minerva application at this point, and you will still be logged in as the admin user you created for Girder.
 
+
+## Option 2: Deploy with ansible-container
+Set the three Girder environment variables in ansible/variables.yml to the correct values for a
+set of working Girder credentials.  This will be the account that Minerva uses to connect with
+Girder.
+    ansible-galaxy install girder.girder
+    export DOCKER_CLIENT_TIMEOUT=600
+    ansible-container build
+    ansible-container run
```

Comment:
You might want to mention installing ansible-container.  I think `pip install ansible-container[docker]` worked for me.